orchagent/bfdorch.cpp

```diff
@@ -517,6 +518,14 @@ bool BfdOrch::create_bfd_session(const string& key, const vector<FieldValueTuple
         attr.id = SAI_BFD_SESSION_ATTR_DST_MAC_ADDRESS;
         memcpy(attr.value.mac, dst_mac.getMac(), sizeof(sai_mac_t));
         attrs.emplace_back(attr);
+
+        attr.id = SAI_BFD_SESSION_ATTR_USE_NEXT_HOP;
+        attr.value.booldata = true;
```

Comment:
i'm confused, default is  'false' and not use NEXTHOP attribute.

Reply:
Here it is under the condition of `alias != "default"`.